src/PlantBundle/Entity/Plant.php

```diff
@@ -18,16 +21,66 @@ class Plant
      * @ORM\GeneratedValue(strategy="AUTO")
      */
     private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="name", type="string", length=255)
+     */
+    private $name;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="place", type="string", length=255, nullable=true)
+     */
+    private $place;
+
     /**
-     * @var boolean
+     * @var string
      *
-     * @ORM\Column(name="is_watered", type="boolean", nullable=true)
+     * @ORM\Column(name="description", type="string", length=255, nullable=true)
      */
-    private $isWatered;
+    private $description;
+
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(name="dateLastWatered", type="datetime", nullable=true)
+     */
+    private $dateLastWatered;
+
+    /**
+     * @var float
+     *
+     * @ORM\Column(name="amount", type="float")
+     */
+    private $amount;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="frequency", type="integer")
+     */
+    private $frequency;
```

Comment:
`frequency` and `frequencyPerDay` can be the same field. Just use one `frequency` and the `frequencyType`, for example. It can be enumeration with options like "times a day", "every x days"